### PR TITLE
feat: run rules

### PR DESCRIPTION
**Pull request by vlalanne**
New keyword introduced in gherkin 6: https://cucumber.io/docs/gherkin/reference/#rule

```gherkin
Feature: math operations
  Rule: add things
    Scenario: add two digits
      When I add 1 and 2
      Then the result should equal 3
```

### CODE REVIEW

File: gobdd.go [hunk 1]
```diff
@@ -642,6 +668,16 @@ func (s *Suite) findStepDef(text string) (stepDef, error) {
 	return sd, nil
 }
 
+func (s *Suite) skipContainer(containerTags []*msgs.Tag) bool {
```
Comment (bkielbasa) [inline gobdd.go:671]: this name is confusing. I'd call this function `shouldSkipStep` or `shouldIgnoreStep`. WDYT?

**Review by bkielbasa — COMMENTED** [review]
LGTM but I'd rename the function I commented and we should be good. Thanks for the PR, BTW! Looks awesome :)

File: gobdd.go (hunk 1 above)
Comment (vlalanne) [reply gobdd.go]: Changed the name to be more explicit.

**Comment by bkielbasa** [comment]
thanks for your contribution!

**Pull request merged**